src/detectors/typeChecker/index.ts

```diff
@@ -212,11 +217,20 @@ export class TsProjectDetector extends ProjectBasedDetector {
 			});
 
 			// Rewrite fs-paths to virtual paths
-			resourcePaths = allResources.map((res: Resource) => {
+			resourcePaths = [...allResources, ...allTestResources].map((res: Resource) => {
 				if (absoluteFilePaths.includes(res.getSourceMetadata().fsPath)) {
 					return res.getPath();
 				}
-			}).filter(($: string | undefined) => $);
+			})
+				.filter(($: string | undefined) => $)
+				.map((res) => {
```

Comment:
Move into the map function three lines above the reduce the number of iterations over this possibly large array

Reply:
Done